packages/lime-proto-anygw/Makefile

```diff
@@ -23,7 +23,7 @@ define Package/$(PKG_NAME)
 	URL:=http://libremesh.org
 	DEPENDS:=+dnsmasq-dhcpv6 @(!PACKAGE_dnsmasq) +ebtables +libuci-lua \
 		+lime-system +lua +kmod-ebtables +kmod-macvlan \
-		+shared-state +shared-state-dnsmasq_leases
+		+shared-state +shared-state-dnsmasq_leases +kmod-ebtables-ipv6
```

Comment:
Didn't you fixed this in another PR? here it seems unrelated

Reply:
That kernel module is needed both by anygw (see #599) and Babeld in case of no VLAN and presence of Batman.

Reply:
Gosh, now I noticed that this file was actually not pertaining to this PR, sorry.